source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRInputDateRenderer.mm

```diff
@@ -34,8 +34,24 @@ - (UIView *)render:(UIView<ACRIContentHoldingView> *) viewGroup
     std::shared_ptr<BaseCardElement> elem = [acoElem element];
     std::shared_ptr<BaseInputElement> dateInput = std::dynamic_pointer_cast<BaseInputElement>(elem);
     ACRDateTextField *dateField = [[ACRDateTextField alloc] initWithTimeDateInput:dateInput dateStyle:NSDateFormatterShortStyle];
-
-    [viewGroup addArrangedSubview: dateField];
+    
+    if(elem->GetHeight() == HeightType::Stretch){
+        ACRColumnView *inputContainer = [[ACRColumnView alloc] init];
+        [inputContainer addArrangedSubview: dateField];
+        
+        // Add a blank view so the input field doesnt grow as large as it can and so it keeps the same behavior as Android and UWP
+        UIView *blankTrailingSpace = [[UIView alloc] init];
+        blankTrailingSpace.backgroundColor = UIColor.clearColor;
+        
+        [blankTrailingSpace setContentHuggingPriority:UILayoutPriorityDefaultHigh forAxis:UILayoutConstraintAxisVertical];
```

Comment:
why is DefaultHigh used? is blankTrailingSpace intended to fill the available space?

Reply:
blankTrailingSpace is the way I found to avoid the UITextField from using all the space it had available and making it not look like the Android and UWP versions

This is how it looks like with the blankTrailingSpace
![simulator screen shot - iphone 7 - 2018-06-11 at 16 04 24](https://user-images.githubusercontent.com/35784165/41261627-70c68c4e-6d91-11e8-9a12-1f9103e0589d.png)


While this one is the version without the blankTrailingSpace
![simulator screen shot - iphone 7 - 2018-06-11 at 16 09 11](https://user-images.githubusercontent.com/35784165/41261704-cedc5282-6d91-11e8-905e-730a5633e29c.png)